src/index.js

```diff
@@ -3,7 +3,8 @@ import ReactDOM from "react-dom/client";
 import { BrowserRouter } from "react-router-dom";
 import reportWebVitals from "./reportWebVitals";
 
-import store from "./redux/state";
+// import store from "./redux/state";
```

Comment:
Do we need this import?

Reply:
for better understanding of what happens with state